tox.ini

```diff
@@ -3,6 +3,10 @@ skipsdist = true
 isolated_build = true
 envlist = lint, coverage
 
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
+
```

Comment:
WHy do we ignore the deprecation ?

Reply:
Some of the tests still use the old API, for example: `path` instead of `default_path`

Reply:
👍🏻